server/src/setup/database/mikro-db.ts

```diff
@@ -48,7 +50,7 @@ class CustomNamingStrategy extends AbstractNamingStrategy implements NamingStrat
 }
 
 const clientUrl = process.env.DATABASE_URL
-// Heroku's postgres service self-signs SSL certificates,
+// Heroku's postgres service self-signs SSL certificates (whatever that means),
```

Comment:
we know what SSL certs are though!

Reply:
lol good catch, this is what happens when we check in code from a branch that is 1.5 weeks behind main